### DISCUSSION TITLE
Screen sticks on Creality logo after Guppy install

### DISCUSSION THREAD

**Discussion by Myklt8** [Guilouz/Creality-Helper-Script-Wiki#550] [Issues]
My K1 Max is stuck on the Creality splash screen after I installed Guppy screen. It worked to do all the configuration, but then got stuck. I can still print through mainsail. I tried to uninstall and it kicked me out of the script. Now it won't let me ssh into the printer, and I can't do a factory reset because of the screen. Any idea where to start.

**Comment by dthreee** [comment]
Having this trouble after installing Git backup. I might have mistyped the token but, like you I can't even get in to factory reset the printer.

**Comment by wepiii** [comment]
I had this EXACT sam issue and was pulling my hair out. The only way I could fix it was to restore the firmware to default, which is difficult without the screen. It can however be done by following the "Reset from WebSocket" instructions located here: https://guilouz.github.io/Creality-Helper-Script-Wiki/firmwares/reset-factory-settings/

**Comment by FaintandFuzzies** [comment]
I'm having the same exact issue, but the reset is not working and can't log into root.
